ResourceStatusSystem/PolicySystem/Actions/EmailAction.py

```diff
@@ -76,7 +76,7 @@ def run(self):
       elif not siteName['Value']:
         siteName = "Unassigned Resources"
       else:
-        siteName = siteName['Value'][0]
+        siteName = siteName['Value'] if isinstance(siteName['Value'], basestring) else siteName['Value'][0]
```

Comment:
isnt' it simpler to take the first of the list in `getSitesForSE` a bit higher ? Then you don't need any condition here

Reply:
Nah, it's more modifications.